src/site/filters/liquid.js

```diff
@@ -368,6 +368,10 @@ module.exports = function registerFilters() {
     return deepObj;
   }
 
+  liquid.filters.getArrayItemByIndex = (arr, idx) => {
+    return arr && arr[idx] ? arr[idx] : '';
+  };
```

Comment:
There is already a [`get`](https://github.com/leizongmin/tinyliquid/blob/cebcb26c3839e725cf0469dccc0073799902a020/README_en.md#arrays-and-objects) filter that might do what we need? It works to get dynamic keys out of object, thinking it might work for array indexes as well?

Reply:
let me check it out...

Reply:
you are correct ... `getValueFromObjPath` works for arrays as well... so I added in a comment above `getValueFromObjPath` to indicate it works for arrays also

Reply:
Oh cool, thank you 😄